dev_tools/packaging/pypirc_template

```diff
@@ -0,0 +1,8 @@
+[pypi]
+username = __token__
+password = ${CIRQ_PYPI_TOKEN}
+
+[testpypi]
+username = __token__
+password = ${CIRQ_TEST_PYPI_TOKEN}
+
```

Comment:
nit - we can delete the trailing blank line here.

Reply:
done